addons/website/views/website_templates.xml

```diff
@@ -912,26 +912,30 @@
             <div class="oe_structure">
                 <h1 class="container mt32"><t t-esc="status_code"/>: <t t-esc="status_message"/></h1>
             </div>
-            <t t-if="editable or request.debug">
+            <t t-if="request.debug">
                 <t t-call="website.http_error_debug"/>
             </t>
         </div>
     </t>
 </template>
 
+<template id="error_message">
+    <p t-if="error_message">
+        <strong>Error message:</strong>
+        <pre t-esc="error_message"/>
+    </p>
+</template>
+
 <template id="http_error_debug">
     <div class="container accordion mb32 mt32" id="debug_infos">
-        <div class="card" t-if="exception">
+        <div class="card" t-if="error_message or website_controller">
```

Comment:
No idea what this is but website_controller is never defined? To verify the name is not crafted in code or why it is needed.